R/contingencytables.R

```diff
@@ -794,11 +818,35 @@ ContingencyTables <- function(jaspResults, dataset, options, ...) {
         proportions.matrix[,] <- ""
       }
 
+      unstandardized.residuals.matrix <- counts.matrix - chisqResults$expected
+
+      if (isTryError(unstandardized.residuals.matrix)) {
+        unstandardized.residuals.matrix    <- counts.matrix
+        unstandardized.residuals.matrix[,] <- ""
+      }
+
+      pearson.residuals.matrix <- chisqResults$residuals
+
+      if (isTryError(pearson.residuals.matrix)) {
+        pearson.residuals.matrix    <- counts.matrix
+        pearson.residuals.matrix[,] <- ""
+      }
+
+      standardized.residuals.matrix <- chisqResults$stdres
+
+      if (isTryError(standardized.residuals.matrix)) {
+        standardized.residuals.matrix    <- counts.matrix
+        standardized.residuals.matrix[,] <- ""
+      }
```

Comment:
This the same as before... You can check whether `chisqResults` is try error once, if not, then extract all these matrices from the result object, otherwise make all the empty tables.